blog/content/post/2020-04-30-ipxe-https.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Enabling HTTPS support in iPXE
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2020-04-30
+archives: "2020"
+
+tags:
+  - firmware
+  - security
+  - coreboot
+categories:
+  - Firmware
+  - Security
+  - coreboot
+  - iPXE
+
+---
+
+Very often it happens that we do not have any burned installation image for our
+desired system at hand. Burning such an image, onto USB for example, can take
+several minutes. When someone gets a brand new hardware, they would like to use
+it as soon as possible, almost like a child with new toy. But how to save those
+few important minutes? The only thing that comes to the mind is network
+booting. In this article I will show you how easy it is to boot from network
+using iPXE on top of coreboot and SeaBIOS. Additionally I will show booting
+from HTTPS.
```

Comment:
@miczyg1 please explain that below method use mainline coreboot and obtained firmware is different then coreboot distribution made by 3mdeb for PC Engines.